### DISCUSSION TITLE
[New Feature] Buildings Pause Repairing when out of Funds/Cash/Money/Resources

### DISCUSSION THREAD

**Discussion by Onidotmoe** [Vinifera-Developers/Vinifera#1347] [Feature Requests]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, videos, links, etc)*.

### Description

When you set buildings to be repaired and you run out of money, they will get unmarked for repairing and you have to mark them again, it would be better if they just paused their repairing until money came in again or until the player has unmarked them for repairing manually.

**Comment by Onidotmoe** [comment]
This is actually how it works in C&C3 btw.

**Comment by JoyfulShush** [comment]
Duplicate of #1256
